hudi-client/hudi-java-client/src/test/java/org/apache/hudi/client/TestJavaHoodieBackedMetadata.java

```diff
@@ -2763,10 +2762,6 @@ private void validateMetadata(HoodieJavaWriteClient testClient, Option<String> i
       // Metadata table is MOR
       assertEquals(metadataMetaClient.getTableType(), HoodieTableType.MERGE_ON_READ, "Metadata Table should be MOR");
 
-      // Metadata table is HFile format
-      assertEquals(metadataMetaClient.getTableConfig().getBaseFileFormat(), HoodieFileFormat.HFILE,
-          "Metadata Table base file format should be HFile");
-
       // Metadata table has a fixed number of partitions
```

Comment:
Why remove this check

Reply:
Going forward we'll have to remove this check as we can have multiple file formats even in metdata table when we support certain secondary indexes in other than HFile format. This check also did not add much value anyway.